static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -0,0 +1,127 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {getGroupDetailsQueryData} from 'sentry/views/issueDetails/utils';
+
+import type {TimelineEvent} from './useTraceTimelineEvents';
+
+interface TraceIssueEventProps {
+  event: TimelineEvent;
+}
+
+export function TraceIssueEvent({event}: TraceIssueEventProps) {
+  const organization = useOrganization();
+  const {projects} = useProjects({
```

Comment:
There is a `useProjectFromSlug` hook you could use here instead, but it doesn't matter much since it does the same thing